source/systems/gaea_user_guide.rst

```diff
@@ -91,2113 +93,1545 @@ points at Atlanta and Chicago.
-have -mixed on the end.
-
-Cray Compiler Wrappers
-----------------------
-
-traditional compiler invocation commands. The wrappers call the
-Cray provides a number of compiler wrappers that substitute for the
-appropriate compiler, add the appropriate header files, and link
-against the appropriate libraries based on the currently loaded
-programming environment module. To build codes for the compute nodes,
-you should invoke the Cray wrappers via:
-
-- cc To use the C compiler
-- CC To use the C++ compiler
-- ftn To use the FORTRAN 90 compiler
-
-PrgEnv-pgi is the default module when you login to Gaea.
-These wrappers are provided by PrgEnv-[intel|gnu|pgi|cray] modules.
-
-Compiling and Node Types
-------------------------
-
-Cray systems are comprised of different types of nodes:
-
-- Login nodes running traditional Linux
-- Batch nodes running traditional Linux
-- Compute nodes running the Cray Node Linux (CNL) microkernel
-  - Your code will run on these nodes.
+Valgrind4hpc is a Valgrind-based debugging tool to aid in the detection of
+memory leaks and errors in parallel applications. Valgrind4hpc aggregates any
+duplicate messages across ranks to help provide an understandable picture of
+program behavior. Valgrind4hpc manages starting and redirecting output from
+many copies of Valgrind, as well as deduplicating and filtering Valgrind
+messages. If your program can be debugged with Valgrind, it can be debugged
+with Valgrind4hpc.
 
-.. warning::
+Valgrind4hpc is available on Gaea under all compiler families:
 
-  Always compile on the login nodes. Never compile on the batch nodes.
+.. code::
 
-.. note::
+    module load valgrind4hpc
 
-  Gaea also has LDTN and RDTN nodes. These are for combining model
-  output (LDTN) and data transfer (RDTN) only, not compiling. They are
-  not Cray nodes.
+Additional information about Valgrind4hpc usage can be found in the `HPE Cray
+Programming Environment User Guide
+<https://support.hpe.com/hpesc/public/docDisplay?docId=a00115110en_us&page=Debug_Applications_With_valgrind4hpc_To_Find_Common_Errors.html>`__.
 
-**Compiling for Compute Node**
 
-Cray compute nodes are the nodes that carry out the vast majority of
-computations on the system. Compute nodes are running the CNL
-microkernel, which is markedly different than the OS running on the
-login and batch nodes. Your code will be built targeting the compute
-nodes. All parallel codes should run on the compute nodes. Compute
-nodes are accessible only by invoking aprun within a batch job. To
-build codes for the compute nodes, you should use the Cray compiler
-wrappers.
+*********
+Profiling
+*********
 
-.. note::
+HPE Performance Analysis Tools
+==============================
 
-  We highly recommend that the Cray-provided cc, CC, and ftn compiler
-  wrappers be used when compiling and linking source code for use on
-  the compute nodes.
+The HPE Performance Analysis Tools (PAT), formerly CrayPAT, are a suite of
```

Comment:
(PAT), formerly CrayPAT, are a suite of utilities that enable users to capture and analyze performance data generated during program execution. 

Either ."..(PAT) is a suite of utilities"
or just say
The HPE Performance Analysis Tools (PAT), formerly CrayPAT, enable users

Reply:
With the initialization and the added phrase, I agree the subject and verb agreement feels off.  However, it isn't.  I removed the initialization and phrase.